docs/data/sdks/next-gen-sdks.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Next Generation SDKs
+description: Introduction to Next Generation SDK, common methods and benefits
+---
+
+This article helps you remove confusion around various Amplitude Analytics SDKs and introduces the Next Generation SDKs.
+
+The **Next Generation (next-gen) SDKs** is a group of Amplitude Analytics SDKs that share the same architecture and interfaces. 
+
+Amplitude puts an SDK into maintenance status and under the "**Maintenance SDKs**" section in the left bar if another same-platform next-gen SDK is available, for example, Browser, Android, React Native, and Node.js. Maintenance SDKs only receives bug fixes until deprecation. It's strongly recommended to adopt next-gen SDKs to take advantage of their benefits. To migrate from a maintenance SDK, refer to the migration guide under each next-gen SDK documentation. 
+
+!!!note 
+    There are still some SDKs that aren't nex-gen while also not in maintenance, for example Flutter, Java, and iOS. 
+
+Refer to the [SDK status table](./#sdk-status-table) to check whether a SDK is next-gen or in maintenance. 
+
+## Architecture
+
+Next-gen SDKs share the same architecture and interfaces across platform. Refer to each SDK full documentation for more details. 
+
+![Next-gen Architecture](/../assets/images/data-next-gen-sdk-architecture.png)
+
+### Client
+
+Amplitude Client stores configurations and is also the entry point for developers to track events. 
+
+Before instrument, you must initialize a SDK using the API key for your Amplitude project, which creates an Amplitude Client. 
+
+```python 
+# Next-gen Python SDK example: initialize a Client
+
+from amplitude import Amplitude
+
+amplitude = Amplitude('API_KEY')
+```
```

Comment:
Browser TS is the most heavily used SDK. Can you please update the examples to be for Browser TS.

Reply:
+1

Reply:
Updated examples to TS